intellij-plugin-verifier/build.gradle.kts

```diff
@@ -129,3 +130,9 @@ tasks {
   }
 }
 
+changelog {
+  version = projectVersion.also { println(it) }
```

Comment:
println?

Reply:
Leftover, removed as this is not necessary.